english/marshmallow.md

```diff
@@ -20,6 +20,9 @@ The type of extreme mortal danger that so often accompanies confectionary inspir
 Quickly surveying your immediate surrounds, you notice a garishly dressed, 
 vacant-eyed young man lounging on a nearby sofa.
 
+[pinch yourself to see if this is a dream](./is-this-a-dream/is-this-a-dream.md)
+[remember that you are kal-el, also known as super man, and fly the hell out of this stupid ass candy house](./remember-you-are-superman/remember-you-are-superman.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.